t/t0009-reset.t

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+test_description='Test Powerman reset control'
+
+. `dirname $0`/sharness.sh
+
+powermand=$SHARNESS_BUILD_DIRECTORY/src/powerman/powermand
+powerman=$SHARNESS_BUILD_DIRECTORY/src/powerman/powerman
+vpcd=$SHARNESS_BUILD_DIRECTORY/test/vpcd
+vpcdev=$SHARNESS_TEST_SRCDIR/../etc/devices/vpc.dev
+
+# Use port = 11000 + test number
+# That way there won't be port conflicts with make -j
+testaddr=localhost:11009
+
+test_expect_success 'create test powerman.conf' '
+	cat >powerman.conf <<-EOT
+	include "$vpcdev"
+	listen "$testaddr"
+	device "test0" "vpc" "$vpcd |&"
+	node "t[0-15]" "test0"
+	EOT
+'
+test_expect_success 'start powerman daemon and wait for it to start' '
+	$powermand -c powerman.conf -f &
+	echo $! >powermand.pid &&
+	$powerman --retry-connect=100 --server-host=$testaddr -q >/dev/null
+'
+test_expect_success 'powerman -r t1 works' '
+	$powerman -h $testaddr -r t1 >reset1.out &&
+	echo "Command completed successfully" >reset1.exp &&
+	test_cmp reset1.exp reset1.out
+'
+test_expect_success 'powerman -r t[3-5] works' '
+	$powerman -h $testaddr -r t[3-5] >reset2.out &&
+	echo "Command completed successfully" >reset2.exp &&
+	test_cmp reset2.exp reset2.out
+'
```

Comment:
not ncessarily for this PR as you're just converting old tests, but should we be checking on/off before and after doing these?

Reply:
No reset doesn't affect on/off.  It's an optional hardware reset (like press the reset button).